src/gbd_mapping/base_template.py

```diff
@@ -97,18 +97,18 @@ def __init__(self,
 
 class Tmred(GbdRecord):
     """Container for theoretical minimum risk exposure distribution data."""
-    __slots__ = ('distribution', 'min', 'max', 'inverted', )
+    __slots__ = ('distribution', 'inverted', 'min', 'max', )
 
     def __init__(self,
                  distribution: str,
-                 inverted: bool, 
+                 inverted: bool,
                  min: scalar = None,
-                 max: scalar = None,):
```

Comment:
curious: does anyone know of a use case for leaving the trailing comma in a function signature?

Reply:
There is a style argument that some people will have, but isn't important.  The practical thing is that it is an easier thing to code to autogenerate since it remove one of your edge cases (args are always followed by `,` rather than args are always followed by `,` except when it's the last arg.)